back/src/Guide/Db/Schema.hs

```diff
@@ -0,0 +1,146 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes       #-}
+
+
+module Guide.Db.Schema where
+
+import Imports
+
+import Hasql.Session (Session)
+import NeatInterpolation
+import Data.Either (lefts)
+import System.IO (stderr, hPrint)
+-- import Hasql.Statement (Statement(..))
+import Hasql.Connection (Connection, Settings)
+
+import qualified Hasql.Session as HS
+-- import qualified Hasql.Decoders as HD
+-- import qualified Hasql.Encoders as HE
+-- import qualified Data.ByteString as B
+import qualified Hasql.Connection as HC
+
+
+main :: IO ()
+main = do
+  conn <- connection
+  dbCreate conn
+
+connection :: IO Connection
+connection = do
+  either (error . show) pure =<< HC.acquire connectionSettings
+
+connectionSettings :: Settings
+connectionSettings = HC.settings "localhost" 5432 "postgres" pass "guide"
+
+pass :: ByteString
+pass = ""
+
+dbCreate :: Connection -> IO ()
+dbCreate conn = do
+  result <- mapM (\s -> HS.run s conn) sessionList
+  mapM_ (hPrint stderr) $ lefts result
+
+sessionList :: [Session ()]
+sessionList =
+  [ createTypeTraitType
+  , createDbCategories
+  , createDbItems
+  , createDbTraits
+  , createDbUsers
+  , createDbEdits
+  ]
+
+createTypeTraitType :: Session ()
+createTypeTraitType = HS.sql "CREATE TYPE trait_type AS ENUM ('pro', 'con');"
+
+createDbTraits :: Session ()
```

Comment:
If you decide not to rename the function, can you provide reasoning for why "db" is better? My reasoning for why "table" is better is that Postgres has [`CREATE DATABASE`](https://www.postgresql.org/docs/9.0/sql-createdatabase.html) as well and it's a different thing from `CREATE TABLE`.

Reply:
ops, I missed that comment.